src/app/hooks/useSendContractTx.ts

```diff
@@ -43,7 +43,7 @@ export interface SendTxResponseInterface extends ResetTxResponseInterface {
     args: any[],
     config?: TransactionConfig,
     options?: TransactionOptions,
-  ) => void;
+  ) => Promise<boolean>;
```

Comment:
Just as a headsup this will clash with the changes I made for meta transactions, where I opted to change it to Promise<string | void>, where either a transactions is returned or not.

Reply:
Changed to `string | void` too, makes more sense to have hash than just boolean.